example/src/CostCalculatorWithPremiumBenefits.tsx

```diff
@@ -26,6 +27,166 @@ const estimationOptions = {
   enableCurrencyConversion: true,
 };
 
+const Layout = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <div className="premium-benefits-wrapper">
+      <div className="premium-benefits-container">{children}</div>
+    </div>
+  );
+};
+
+const Header = () => {
+  return (
+    <div className="premium-benefits-header">
+      <h1>Cost calculator</h1>
+      <p>Estimate the cost to hire someone through Remote</p>
+      {/** TODO: Add a zendesk link that opens a Zendesk drawer */}
+      {/* <a href="https://remote.com/premium-benefits">Learn more</a> */}
+    </div>
+  );
+};
+
+const ActionToolbar = ({
+  onReset,
+  onExportPdf,
+  onCSVExport,
+}: {
+  onReset: () => void;
+  onExportPdf: () => void;
+  onCSVExport: () => void;
+}) => {
+  return (
+    <div className="premium-benefits-action-toolbar">
+      <div>
+        <button
+          className="premium-benefits-action-toolbar__button"
+          onClick={onReset}
+        >
+          Reset estimate
+        </button>
+      </div>
+      <div className="flex gap-2">
+        <button
+          className="premium-benefits-action-toolbar__button premium-benefits-action-toolbar__button--secondary"
+          onClick={onCSVExport}
+          disabled
+        >
+          Export as CSV
+        </button>
+        <button
+          className="premium-benefits-action-toolbar__button premium-benefits-action-toolbar__button--secondary"
+          onClick={onExportPdf}
+        >
+          Export as PDF
+        </button>
+        <button
+          className="premium-benefits-action-toolbar__button premium-benefits-action-toolbar__button--primary"
+          disabled
+        >
+          Add estimate
+        </button>
+      </div>
+    </div>
+  );
+};
+
+const InitialForm = ({
+  onSubmit,
+  onError,
+  onSuccess,
+}: {
+  onSubmit: (payload: CostCalculatorEstimationSubmitValues) => void;
+  onError: (error: EstimationError) => void;
+  onSuccess: (response: CostCalculatorEstimateResponse) => void;
+}) => {
+  return (
+    <>
+      <Header />
+      <CostCalculatorFlow
+        estimationOptions={estimationOptions}
+        options={{
+          jsfModify: {
+            fields: {
+              country: {
+                title: 'Employee country',
+                description:
+                  'Select the country where the employee will primarily live and work',
+              },
+              currency: {
+                title: 'Employer billing currency',
+                description:
+                  'Select the currency you want to be invoiced in for this employee’s services.',
+              },
+              salary: {
+                title: "Employee's annual salary",
```

Comment:
very minor neatpick: there's some inconsistency on quotes usage, you use forward tick below, while use ' here